apps/web/src/pages/DirectivePage/Resources.tsx

```diff
@@ -0,0 +1,426 @@
+          })}
+        </Heading>
+        <div
+          data-h2-display="base(grid)"
+          data-h2-grid-template-columns="base(1fr) p-tablet(repeat(2, minmax(0, 1fr))) l-tablet(repeat(3, minmax(0, 1fr)))"
+          data-h2-gap="base(x1)"
+          data-h2-margin="base(x1, 0, 0, 0) p-tablet(x2, 0, 0, 0)"
+        >
+          <Card
+            color="black"
+            title={intl.formatMessage({
+              defaultMessage: "Digital initiative managers",
+              id: "Tvsi5A",
+              description: "Title for group-specific resource card",
+            })}
+            noPadding
+          >
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-justify-content="base(space-between)"
+              data-h2-height="base(100%)"
+            >
+              <div>
+                <img
+                  src={managers}
+                  alt=""
+                  data-h2-display="base(none) p-tablet(block)"
+                />
+
+                <p data-h2-margin="base(x1)">
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "These resources are designed to support digital initiative managers and leads in fulfilling their responsibilities under the Directive on Digital Talent. This implementation guidance explains why the directive is needed and what is required. Use the decision tree to navigate the reporting requirements.",
+                    id: "Iz8E+y",
+                    description:
+                      "Body message for digital initiative managers section.",
+                  })}
+                </p>
+              </div>
+              <div data-h2-margin="base(0, x1, x2, x1)">
+                <p data-h2-font-weight="base(bold)">
+                  {intl.formatMessage({
+                    defaultMessage: "Implementation guidance for managers",
+                    id: "wJ9hiY",
+                    description:
+                      "label above download guidance for managers pdf and text.",
+                  })}
+                </p>
+                <Link
+                  mode="inline"
+                  color="secondary"
+                  block
+                  external
+                  download
+                  href={
+                    localeState.locale === "en"
+                      ? guidanceManagerPdfEn
+                      : guidanceManagerPdfFr
+                  }
+                  icon={ArrowDownOnSquareIcon}
+                  data-h2-margin="base(x1, 0, x1, 0)"
+                  data-h2-justify-content="base(flex-start)"
+                >
+                  {intl.formatMessage({
+                    defaultMessage: "Download (PDF)",
+                    id: "nJB8I6",
+                    description:
+                      "Button text to download guidance for managers pdf.",
+                  })}
+                </Link>
```

Comment:
So, these links used to include more context. I agree with reducing the visual text present but I think we need the original text as `aria-labels` otherwise AT users may not know what they are downloading.

This applies to all links on these cards.

![Screenshot 2023-12-04 140819](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/dd02fc40-a94f-4196-b180-5a269157010b)

Reply:
Not sure why the static-pages.cy.ts e2e test keeps failing here. It's passing for me locally. Looking into it now.